tests/Integration/inc/classes/subscriber/third-party/plugins/Smush_Subscriber/SmushSubscriberTestCase.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\classes\third_party\plugins\Smush_Subscriber;
+
+use Smush\Core\Settings;
+use WP_Rocket\Subscriber\Third_Party\Plugins\Smush_Subscriber;
+use WPMedia\PHPUnit\Integration\TestCase;
+
+abstract class SmushSubscriberTestCase extends TestCase {
+	protected $subscriber;
+	protected $smush;
+	protected $smush_settings_option_name;
+	protected $smush_settings;
+	protected $smush_lazy_option_name;
+	protected $smush_lazy;
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->subscriber                 = new Smush_Subscriber();
```

Comment:
Pull the subscriber out of the container by using the `apply_filter`. You'll want now that we are injecting options into it.